To: vim_dev@googlegroups.com
Subject: Patch 7.3.246
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.246 (after 7.3.235)
Problem:    Repeating "f4" in "4444" skips one 4.
Solution:   Check the t_cmd flag. (Christian Brabandt)
Files:	    src/search.c


*** ../vim-7.3.245/src/search.c	2011-06-26 05:36:07.000000000 +0200
--- src/search.c	2011-07-15 13:16:49.000000000 +0200
***************
*** 1585,1591 ****
  	/* Force a move of at least one char, so ";" and "," will move the
  	 * cursor, even if the cursor is right in front of char we are looking
  	 * at. */
! 	if (vim_strchr(p_cpo, CPO_SCOLON) == NULL && count == 1)
  	    stop = FALSE;
      }
  
--- 1585,1591 ----
  	/* Force a move of at least one char, so ";" and "," will move the
  	 * cursor, even if the cursor is right in front of char we are looking
  	 * at. */
! 	if (vim_strchr(p_cpo, CPO_SCOLON) == NULL && count == 1 && t_cmd)
  	    stop = FALSE;
      }
  
*** ../vim-7.3.245/src/version.c	2011-07-15 13:09:46.000000000 +0200
--- src/version.c	2011-07-15 13:20:40.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     246,
  /**/

-- 
   GALAHAD turns back.  We see from his POV the lovely ZOOT standing by him
   smiling enchantingly and a number of equally delectable GIRLIES draped
   around in the seductively poulticed room.  They look at him smilingly and
   wave.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
